types/project/CartConfiguration.raml

```diff
@@ -0,0 +1,7 @@
+#%RAML 1.0 DataType
```

Comment:
I would suggest to rename the file to `CartsConfiguration.raml`, because it's more consistent with our docs and our existing SDKs.